### ISSUE TITLE
Support for 0d ndarrays

### ISSUE THREAD

**Issue by antimatter15** [scijs/ndarray-ops#9]
This doesn't seem to work with 0d ndarrays:

    var a = ndarray(new Float32Array(128*128))
    ops.addseq(a.pick(0), 2)

Throws  "SyntaxError: Unexpected token |="

It looks like cwise-compiler (maybe this is a bug which should be filed there instead) is assembling some code which includes a `var` statement with no argument (i.e. "var \n a |= 0").

**Comment by rreusser** [comment]
Hmm… I can see the argument either way. The amount of data is defined as the product of the size of the dimensions. So a shape of `[2, 2]` has four numbers. `[5]` has five numbers. `[]` has no data. So it's a no-op.

So maybe it's that it should do nothing and exit early.

**Comment by mikolalysenko** [comment]
Yeah, I think this is a bug in cwise.  Should open an issue there as well.

**Comment by Jabher** [comment]
I've made a fix: https://github.com/scijs/cwise-compiler/pull/4
already accepted, waiting for npm release to make it work. This issue can be closed right after the release

**Comment by Jabher** [comment]
@rreusser correct, but not strictly correct. Size of array is `shapes.reduce((a,b) => a * b, 1)` which means that <2x2> tensor, which is matrix, has size 4; <2> tensor, which is vector, has size 2.  But <> tensor (here goes mind blown) is __scalar__ of __size 1__. And it is totally fine to use tensor algebra against it.
